src/EditorFeatures/Core/GoOrFind/AbstractGoOrFindNavigationService.cs

```diff
@@ -59,7 +60,7 @@ internal abstract class AbstractGoOrFindCommandHandler<TLanguageService, TComman
     /// the presenter.  In that case, the presenter will notify us that it has be re-purposed and we will also cancel
     /// this source.
     /// </remarks>
-    private readonly CancellationSeries _cancellationSeries = new(threadingContext.DisposalToken);
+    private CancellationTokenSource _cancellationTokenSource = new();
```

Comment:
i restored this to the original logic.  it's acutaly subtly different from a cancellation series in a way i didn't realize.